sfge/src/main/java/com/salesforce/graph/build/InheritanceEdgeBuilder.java

```diff
@@ -85,17 +85,30 @@ private void processVertexInheritance(InheritableSFVertex vertex) {
                     Schema.EXTENSION_OF);
         }
 
-        if (implementedVertices != null) {
-            for (InheritableSFVertex implementedVertex : implementedVertices) {
-                // It's possible for the implementedVertex to be null if the interface being
-                // implemented is declared in a file
-                // that wasn't scanned.
+        if (!implementedVerticesByName.isEmpty()) {
+            List<String> implementedVertexDefiningTypes = new ArrayList<>();
+            Long myId = vertex.getId();
+            for (String implementedVertexName : implementedVerticesByName.keySet()) {
```

Comment:
As indicated at line 124, the map contains an entry for every interface implemented by the class. If no vertex was found, the name was mapped to null.
This lets us iterate over every interface the class implements, instead of only those with corresponding vertices.